ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/ICoreConstants.java

```diff
@@ -60,6 +60,9 @@ public interface ICoreConstants {
 	/** Constant for the string <code>build.properties</code> */
 	String BUILD_FILENAME_DESCRIPTOR = "build.properties"; //$NON-NLS-1$
 
+	/** Constant for the string <code>pde.bnd</code> */
+	String BND_FILENAME_DESCRIPTOR = "pde.bnd"; //$NON-NLS-1$
```

Comment:
There is already `org.eclipse.pde.internal.core.natures.BndProject.INSTRUCTIONS_FILE` so you should at laest reference it here
```suggestion
	String BND_FILENAME_DESCRIPTOR = BndProject.INSTRUCTIONS_FILE;
```